frontend/javascripts/oxalis/model/accessors/dataset_accessor.js

```diff
@@ -103,6 +103,14 @@ export function getElementClass(dataset: APIDataset, layerName: string): Element
   return getLayerByName(dataset, layerName).elementClass;
 }
 
+export function getDefaultIntensityRangeOfLayer(
+  dataset: APIDataset,
+  layerName: string,
+): [number, number] {
+  const elementClass = getElementClass(dataset, layerName);
+  return elementClass === "uint16" ? [0, 65535] : [0, 255];
```

Comment:
I think this should handle float32, as well. Also did you test this code for RGB layer (aka uint24)?